circle.yml

```diff
@@ -228,6 +228,33 @@ jobs:
             prerelease_flag=$([[ $CIRCLE_TAG =~ ^v[0-9\.]+$ ]] || echo '-prerelease')
             ghr -u ethereum -r cpp-ethereum $prerelease_flag $CIRCLE_TAG ~/package
 
+  # Builds aleth docker image and uploads it to Docker Hub.
+  docker-aleth:
+    docker:
+      - image: circleci/golang
```

Comment:
Why is the base image a circleci/golang?

Reply:
CircleCI had this one in their example: https://circleci.com/docs/2.0/building-docker-images/#example.